src/Components/PostsActivitySection.jsx

```diff
@@ -0,0 +1,102 @@
+import { Flex, Text, Box, Icon, HStack } from "@chakra-ui/react";
+import { chat, thumbsUp, stack } from "../icons/icons";
+import Vector from "../icons/Vector.svg";
+
+function PostsActivitySection() {
+  return (
+    <>
+      <Flex flexDir={"column"} gap={6} padding="1rem">
+        <HStack h={"4rem"} padding="2rem" bg="#fff" gap={"2rem"}>
+          <HStack>
+            <img src={Vector} alt="Posts" />
+            <Text fontSize={"xl"}>Posts</Text>
+          </HStack>
+
+          <HStack>
+            <Icon as={stack} boxSize={"1.7rem"}></Icon>
+            <Text fontSize={"xl"}>Activity</Text>
+          </HStack>
+        </HStack>
+        {/* first post starts here */}
```

Comment:
Use .map() function rather than doing it this way

Reply:
okay